themes/webdriver.io/layout/_partial/footer.ejs

```diff
@@ -4,7 +4,7 @@
         <article class="col2">
             <p>This project is maintained by:</p>
             <a class="twitterLink" href="http://twitter.com/webdriverio" title="WebdriverIO"><img src="/images/twitter-profile.png" alt="@WebdriverIO"></a>
-            <a class="twitterLink" href="http://twitter.com/bromann" title="Christian Bromann"><img src="https://pbs.twimg.com/profile_images/1806512295/profil.png" alt="@bromann"></a>
+            <a class="twitterLink" href="http://twitter.com/bromann" title="Christian Bromann"><img src="https://avatars3.githubusercontent.com/u/731337?v=3&s=72" alt="@bromann"></a>
```

Comment:
Still 58k. Could you upload a JPG instead of PNG. That should make a big difference.